util/missingcrypto.txt

```diff
@@ -806,7 +806,6 @@ OCSP_REQUEST_it(3)
 OCSP_REQUEST_print(3)
 OCSP_REQ_CTX_get0_mem_bio(3)
 OCSP_REQ_CTX_http(3)
-OCSP_REQ_CTX_i2d(3)
```

Comment:
I think the function should also be removed from `missingcrypto111.txt`.

Reply:
no, because this is for master only. i can open a separate PR if the project thinks the documentation should be backported.  (but I would not do the deprecation)